pandas/tests/dtypes/test_dtypes.py

```diff
@@ -1152,3 +1153,13 @@ def test_multi_column_dtype_assignment():
 
     df["b"] = 0
     tm.assert_frame_equal(df, expected)
+
+
+def test_Float64_conversion():
```

Comment:
Could you place this in `pandas/tests/arrays/floating/test_astype.py`

Reply:
Sure, I can do that.